pkg/browsers/console.go

```diff
@@ -26,14 +26,8 @@ var ServiceMap = map[string]string{
 	"":               "console",
 	"ec2":            "ec2/v2",
 	"sso":            "singlesignon",
-	"ecs":            "ecs",
```

Comment:
In line with my suggestion below I'd propose reverting the deleted services from `ServiceMap` and leaving the map unchanged. We can keep adding known services to this so that the only times we're falling back to an arbitrary name is (a) when a new service is introduced that we don't yet handle and (b) when a user has made a typo in the service name.

Reply:
done!